### PR TITLE
Fix: Using new default1 profile when there is not profile.

### PR DESCRIPTION
**Pull request by malaniz**
We need to use a default profile when there is not profile on `use as alias` operation.
https://github.com/decentraland/builder/issues/1214

### CODE REVIEW

File: src/modules/ens/sagas.ts [hunk 1]
```diff
@@ -76,11 +76,9 @@ function* handleSetAlias(action: SetAliasRequestAction) {
   try {
     const client = new CatalystClient(PEER_URL, 'builder')
     const entities: Entity[] = yield client.fetchEntitiesByPointers(EntityType.PROFILE, [address.toLowerCase()])
-    const entity = entities.length > 0 ? entities[0] : null
+    const defaultEntity: Entity = yield getDefaultProfileEntity()
```
Comment (nachomazzara) [inline src/modules/ens/sagas.ts:79]: Even this works, I think that we are using for consistency `yield call(....)`. Am I right? same for the line above.

File: src/modules/ens/sagas.ts [hunk 2]
```diff
@@ -76,11 +76,9 @@ function* handleSetAlias(action: SetAliasRequestAction) {
...
+    const entity = entities.length > 0 ? entities[0] : defaultEntity
```
Comment (nachomazzara) [inline src/modules/ens/sagas.ts:80]: I think that we can avoid the default entity's request when the user already has a profile. 

```typescript
let entity: Entity
if (entities.length > 0) {
    entity = entities[0]
} else {
   entity = yield call(() => getDefaultProfileEntity())
}
```

**Review by nachomazzara — CHANGES_REQUESTED** [review]
🚀

File: src/modules/ens/sagas.ts (hunk 2 above)
Comment (malaniz) [reply src/modules/ens/sagas.ts:80]: It looks better in that way. Fixing! Thanks

File: src/modules/ens/sagas.ts (hunk 1 above)
Comment (malaniz) [reply src/modules/ens/sagas.ts:79]: fixing!

**Pull request merged**